internal/exec/vendor_utils.go

```diff
@@ -634,7 +635,7 @@ func generateSkipFunction(atmosConfig schema.AtmosConfiguration, tempDir string,
 		}
 
 		// If 'included_paths' is not provided, include all files that were not excluded
-		u.LogTrace(fmt.Sprintf("Including '%s'\n", u.TrimBasePathFromPath(tempDir+"/", src)))
+		log.Debug("Including ", "file", u.TrimBasePathFromPath(tempDir+"/", src))
```

Comment:
No need to add white space, log does that for you. 
```suggestion
		log.Debug("Including", "file", u.TrimBasePathFromPath(tempDir+"/", src))
```